base/libgit2/types.jl

```diff
@@ -705,13 +705,19 @@ function getobjecttype(obj_type::Cint)
     end
 end
 
+import Base.securezero!
+
 "Credentials that support only `user` and `password` parameters"
 type UserPasswordCredentials <: AbstractCredentials
     user::AbstractString
```

Comment:
you do occasionally get things coming from SubString or regex matches etc in a few places in the Pkg code, but not sure if that was the motivation here or what